devtools.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency 'rspec',        '~> 3.4.0'
   gem.add_runtime_dependency 'rspec-core',   '~> 3.4.0'
   gem.add_runtime_dependency 'rspec-its',    '~> 1.2.0'
-  gem.add_runtime_dependency 'rake',         '~> 10.4.2'
+  gem.add_runtime_dependency 'rake',         '~> 11.1.x'
```

Comment:
Yeah and I'm saying that we don't *really* care about the patch version because we don't diligently update things like rake when `11.1.3` is released. This is a more conservative flavor of the point I was arguing before: I would like to minimize conflicts and Rake `11.1.2` was released [~36 hours ago](https://github.com/ruby/rake/releases/tag/v11.1.2) so I'd rather this gemspec just read `~> 11.1.x`.